src/audiomixcontroller/DefaultAudioMixController.ts

```diff
@@ -86,13 +103,17 @@ export default class DefaultAudioMixController implements AudioMixController {
     if (this.browserBehavior.hasChromiumWebRTC()) {
       existingAudioElement.srcObject = null;
     }
-    try {
-      // @ts-ignore
-      await existingAudioElement.setSinkId(newSinkId);
-    } catch (error) {
-      this.logger?.error(`Failed to set sinkId for audio element: ${error}`);
-      throw error;
+
+    if (shouldSetSinkId) {
+      try {
+        // @ts-ignore
+        await existingAudioElement.setSinkId(newSinkId);
+      } catch (error) {
+        this.logger?.error(`Failed to set sinkId for audio element: ${error}`);
```

Comment:
Nitpick: Do we want to change this to like we did here in [bindAudioStream](https://github.com/aws/amazon-chime-sdk-js/pull/930/files?file-filters%5B%5D=.json&file-filters%5B%5D=.ts#diff-646676a4d8e0ab238513a20bd428ed47017c8ef824e7a99f69731a5dbf315983L40)

```
/* istanbul ignore else */
if (this.logger) {
  this.logger.error(`Failed to set sinkId for audio element: ${error}`);
}
```

Reply:
The only reason I split it out is because the no-logger case wasn't covered by tests, and it was not worth creating another test case just to exercise the non-presence of a log item if the logger was not provided, which is kinda meaningless.